### DISCUSSION TITLE
After Updating Template from 3.0.11 to 3.0.12 error: modules/mod_menu/tmpl/default_url.php:51

### DISCUSSION THREAD

**Discussion by MichaelMGK** [templaza/astroid-framework#717] [Q&A]
Hello all,  I am using for a website Astroid 3.0.12 and today I update the ONE-Template to 3.0.12. But after update the Website was not reachable. I got an error that the startingpage is not reachable. Backend is useable.

This is the error Message: ERROR 0 - Call to a member function get() on null in /var/www/vhosts/hosting115204xxxxxxx/2022.mariae-himmelfahrt-mh.de/httpdocs/modules/mod_menu/tmpl/default_url.php:51

Any Idea how I can fix it?

**Comment by sonvnn** [comment]
@MichaelMGK Please help me update your template to 3.0.15. You can download it here https://github.com/templaza/astroid-framework/tree/one-template
